### ISSUE TITLE
bottled illusion rune activation can interrupt tp

### ISSUE THREAD

**Issue by SergeyKozyr** [jeffhill/Dota2#154]
### Description

My own tp was interrupted by illusion rune activation after 90 seconds

**Comment by NMOTH** [comment]
This is intended behavior unfortunately . Channeling spells get interrupted by using items or summoning illusions. 
Otherwise people can dodge stuns mid tp by just summoning illusions.

**Comment by isRobertMa** [comment]
I don't think this is a bug, the rune activation mechanic will interrupt tp/channeling stands.
